sbi/analysis/plot.py

```diff
@@ -0,0 +1,136 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+import collections
+from typing import Any, List, Optional, Tuple, Union
+
+import numpy as np
+import torch
+from matplotlib import pyplot as plt
+
+from sbi.utils import conditional_pairplot as utils_conditional_pairplot
+from sbi.utils import pairplot as utils_pairplot
+
+try:
+    collectionsAbc = collections.abc
+except:
+    collectionsAbc = collections
+
+
+def pairplot(
+    samples: Union[
+        List[np.ndarray], List[torch.Tensor], np.ndarray, torch.Tensor
+    ] = None,
+    points: Optional[
+        Union[List[np.ndarray], List[torch.Tensor], np.ndarray, torch.Tensor]
+    ] = None,
+    limits: Optional[Union[List, torch.Tensor]] = None,
+    subset: List[int] = None,
+    upper: Optional[str] = "hist",
+    diag: Optional[str] = "hist",
+    fig_size: Tuple = (10, 10),
```

Comment:
I wanted to change this for a long time. the matplotlib arg is `figsize` not `fig_size`. I think it would be good to have it as in matplotlib. Or would that be too annoying now because it breaks all the plotting scripts of the users?

Reply:
I could not agree more. We can for now add both arguments and give a deprecation warning if the user sets `fig_size`. In the future we can take it out then.